tool/teleport/common/configurator.go

```diff
@@ -126,14 +139,24 @@ func onConfigureDiscoveryBootstrap(flags configureDiscoveryBootstrapFlags) error
 		return trace.Wrap(err)
 	}
 
-	fmt.Printf("Reading configuration at %q...\n\n", flags.config.ConfigPath)
+	if flags.config.DiscoveryService && flags.databaseServiceRole != "" {
+		config := makeDatabaseServiceBootstrapFlagsWithDiscoveryServiceConfig(flags)
+		dbConfigurators, err := configuratorbuilder.BuildConfigurators(config)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		configurators = append(configurators, dbConfigurators...)
+	}
+
+	fmt.Printf("Reading configuration at %q...\n", flags.config.ConfigPath)
```

Comment:
It seems like this line should go above this one

https://github.com/gravitational/teleport/blob/2b39df8fd1853ed314c61d7966827afa87c0e203/tool/teleport/common/configurator.go#L137